.env

```diff
@@ -2,3 +2,4 @@ DHL_UK_BASE_URL=https://services.qa.dhlparcel.co.uk
 DHL_UK_API_KEY=
 DHL_UK_USERNAME=
 DHL_UK_PASSWORD=
+DHL_UK_ACCOUNT=
```

Comment:
Should this be set in a client's environment instead?

Reply:
I just added it so I could obscure in the vcrs

Reply:
Yep, that's fine. It will be up to clients anyway how they choose to pass the account number.